src/Nest/CommonAbstractions/Infer/IndexName/IndexName.cs

```diff
@@ -19,24 +19,25 @@ public class IndexName : IEquatable<IndexName>, IUrlParameter
 
 		private IndexName(string index, string cluster = null)
 		{
-			this.Name = index?.Trim();
-			this.Cluster = cluster?.Trim();
+			this.Name = index;
+			this.Cluster = cluster;
 		}
 		private IndexName(Type type, string cluster = null)
 		{
 			this.Type = type;
-			this.Cluster = cluster?.Trim();
+			this.Cluster = cluster;
 		}
 		private IndexName(string index, Type type, string cluster = null)
 		{
 			this.Name = index;
 			this.Type = type;
-			this.Cluster = cluster?.Trim();
+			this.Cluster = cluster;
 		}
 
 		public static IndexName From<T>() => typeof(T);
 		public static IndexName From<T>(string clusterName) => From(typeof(T), clusterName);
 		private static IndexName From(Type t, string clusterName) => new IndexName(t, clusterName);
+		// TODO private?
```

Comment:
TODO?

Reply:
left it in there to tackle at a later stage (not related to commit).